### PR TITLE
[risk=low][RW-11368] Bump spotless plugin to latest - required for Java 17

### PR DESCRIPTION
**Pull request by jmthibault79**
No functional changes expected.  Best reviewed with "hide whitespace" !

Tested locally with no problems.

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [x] None of the above apply to this change.

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/actionaudit/ActionAuditEvent.kt [hunk 1]
```diff
@@ -11,7 +11,7 @@ data class ActionAuditEvent constructor(
     val targetPropertyMaybe: String? = null,
     val targetIdMaybe: Long? = null,
     val previousValueMaybe: String? = null,
-    val newValueMaybe: String? = null
+    val newValueMaybe: String? = null,
```
Comment (jmthibault79) [inline api/src/main/java/org/pmiops/workbench/actionaudit/ActionAuditEvent.kt:14]: Kotlin formatting changes comprise most of this PR

**Comment by jmthibault79** [comment]
All codacy errors are "function should have better documentation" - agreed, but I'm not planning to do that here.

**Review by freemabd — APPROVED** [review]
LGTM

File: api/src/main/java/org/pmiops/workbench/actionaudit/auditors/AdminAuditor.kt [hunk 2]
```diff
@@ -4,9 +4,17 @@ import org.pmiops.workbench.model.AccessReason
 import org.pmiops.workbench.model.AdminLockingRequest
 
 interface AdminAuditor {
-    fun fireViewNotebookAction(workspaceNamespace: String, workspaceName: String, notebookFilename: String, accessReason: AccessReason)
+    fun fireViewNotebookAction(
+        workspaceNamespace: String,
+        workspaceName: String,
+        notebookFilename: String,
+        accessReason: AccessReason,
```
[the remark this answers is not in the document]
Comment (jmthibault79) [reply api/src/main/java/org/pmiops/workbench/actionaudit/auditors/AdminAuditor.kt:11]: other languages like JavaScript and Go prefer the final `,` as well - this seems to be the current style trend.

**Pull request merged**